tests/spec/core/data-include-spec.js

```diff
@@ -135,18 +135,19 @@ describe("Core — Data Include", () => {
         body: `
           <section id="abstract"></section>
 
-          <section data-include="${include}" data-include-format="markdown"></section>
+          <section id="test" data-include="${include}" data-include-format="markdown"></section>
         `,
       };
       ops.abstract = null;
 
       const doc = await makeRSDoc(ops);
-      const li = doc.querySelector("section li");
+      const li = doc.querySelector("section#test li");
       expect(li).toBeTruthy();
-      expect(li.textContent).toContain("Rose");
-      expect(li.textContent).toContain("Blue");
-      expect(li.textContent).toContain("Red");
-      expect(li.textContent).toContain("wasteland");
+      expect(li.textContent).toBe("Rose");
+      const [p1, p2, p3] = [...doc.querySelectorAll("#test p")];
+      expect(p1.textContent).toContain("Blue");
+      expect(p2.textContent).toContain("Red");
+      expect(p3.textContent).toContain("wasteland");
```

Comment:
See https://github.com/markedjs/marked/issues/1934 (and the fix: https://github.com/markedjs/marked/pull/1936)
cc: @saschanaz

Reply:
The intention here was to wrap them to `li`. It seems marked now requires at least 3-space indentation here.

Reply:
3-space fixes it. Shouldn't be a breaking change for us as we've delegated markdown related changes on marked, which is commonmark compliant (almost).
Thanks!